packages/admin-panel/package.json

```diff
@@ -57,7 +57,7 @@
     "lodash.groupby": "^4.6.0",
     "lodash.keyby": "^4.6.0",
     "lodash.throttle": "^4.1.1",
-    "moment": "^2.22.2",
```

Comment:
Was running in to typescript errors due to using multiple different version of `moment` all over the place. So switched to a standard version used by the `web-config-server`. (Decided to go with that version since there's some logic that relies on that version and didn't want to touch legacy code).

We're deprecating use of moment anyway, so I don't mind us being locked in.